src/lib/style/components/facet_nav/_facet-nav.scss

```diff
@@ -30,7 +30,7 @@
   }
 }
 
-.container-full-width {
+.container-full-width .FacetNav.less{
```

Comment:
Why do these changes fix the issue?  Today when showMoreState='LESS' (or any value), this css applies.  What am I missing @conundrumgirl ?